src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/ITypeSymbolExtensions.cs

```diff
@@ -739,5 +742,40 @@ public static ITypeSymbol WithNullableAnnotationFrom(this ITypeSymbol type, ITyp
 
             return symbol;
         }
+
+        public static ImmutableArray<IMethodSymbol> GetLocalFunctionSymbols(
+            this ITypeSymbol? typeSymbol,
+            ISyntaxFactsService syntaxFacts,
+            Compilation compilation,
+            CancellationToken cancellationToken)
+        {
+            if (typeSymbol is null || compilation.Language != LanguageNames.CSharp)
```

Comment:
❓ is it worth avoiding enumeration of members gated on the compilation language here?

Reply:
yes.  this check seems totally reasonable to me :)

Reply:
oh... this is a generic helper now? hrmm... we can always keep this here, and remove later if a problem.  but add comment for now that VB doesn't have local functions.

Reply:
alternatively, move this to ISemanticFactsService.  then teh VB impl can no-op with an appropriate comment.

Reply:
I like this. I forgot about ISemanticFactsService.